cmd/ebrelayer/txs/relayToCosmos.go

```diff
@@ -43,49 +48,55 @@ func RelayToCosmos(cdc *codec.Codec, moniker, password string, claims []types.Et
 	// Prepare tx
 	txBldr, err := utils.PrepareTxBuilder(txBldr, cliCtx)
 	if err != nil {
-		log.Println("error building tx: ", err)
-		log.Println("tx buidler response on error: ", txBldr)
+		sugaredLogger.Errorw("failed to get tx builder.",
+			errorMessageKey, err.Error(),
+			"transaction builder", txBldr)
 		return err
 	}
 
-	log.Printf("ebrelayer RelayToCosmos sequenceNumber is %d from tx builder\n", txBldr.Sequence())
+	sugaredLogger.Infow("relay sequenceNumber from builder.",
+		"next sequence number", txBldr.Sequence())
 
 	// If we start to control sequence
 	if nextSequenceNumber > 0 {
 		txBldr.WithSequence(nextSequenceNumber)
-		log.Println("txBldr.WithSequence(nextSequenceNumber) passed")
+		sugaredLogger.Infow("txBldr.WithSequence(nextSequenceNumber) passed")
 	}
-	
+
 	log.Println("building and signing")
 	// Build and sign the transaction
 	txBytes, err := txBldr.BuildAndSign(moniker, password, messages)
 	if err != nil {
-		log.Printf("ebrelayer RelayToCosmos error 1 is %s\n", err.Error())
+		sugaredLogger.Errorw("failed to sign transaction.",
+			errorMessageKey, err.Error())
 		return err
 	}
-	
+
 	log.Println("built tx, now broadcasting")
 	// Broadcast to a Tendermint node
 	res, err := cliCtx.BroadcastTxAsync(txBytes)
 	if err != nil {
-		log.Printf("ebrelayer RelayToCosmos error 2 is %s\n", err.Error())
+		sugaredLogger.Errorw("failed to broadcast tx to sifchain.",
+			errorMessageKey, err.Error())
 		return err
 	}
 	log.Println("Broadcasted tx without error")
 
 	if err = cliCtx.PrintOutput(res); err != nil {
-		log.Printf("ebrelayer RelayToCosmos error 3 is %s\n", err.Error())
+		sugaredLogger.Errorw("failed to print out result.",
+			errorMessageKey, err.Error())
 		return err
 	}
-	log.Println("printed tx output")
 
 	// start to control sequence number after first successful tx
 	if nextSequenceNumber == 0 {
 		setNextSequenceNumber(txBldr.Sequence() + 1)
 	} else {
 		incrementNextSequenceNumber()
 	}
-	log.Printf("ebrelayer RelayToCosmos nextSequenceNumber is %d after tx\n", nextSequenceNumber)
+	sugaredLogger.Infow("relay next sequenceNumber from memory.",
+		"next sequence number", nextSequenceNumber)
```

Comment:
It would be great if instead of having a key in the JSON logs that is `next sequence number`, it is a single word key like this `nextSequenceNumber`